static/i18n/en/app.json

```diff
@@ -1583,6 +1591,18 @@
     },
     "FeeTooHigh": {
       "title": "Network fees represent more than 10% of the amount"
+    },
+    "StellarWrongMemoFormat": {
+      "title": "Memo format is wrong"
+    },
+    "StellarMinimumBalanceWarning": {
+      "title": "A minimum balance is required to be kept in a Stellar account"
+    },
+    "StellarNewAccountMinimumTransaction": {
+      "title": "A new account requires at least 1 XLM to be activated"
```

Comment:
this must not be in the wording. wording should be 

```
"A new account requires at least {{amount}} to be activated
```

which interestingly makes this wording generic. to me we could drop Stellar prefix?


but also, isn't this already existing for another coin @juan-cortes  ?